domains/seven.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "7op3",
+           "email": "134126156+7op3@users.noreply.github.com",
+           "discord": "882124409945587773"
+        },
+    
+        "record": {
+            "CNAME": "7op3.github.io"
```

Comment:
Little content shown on the site.

Reply:
I've updated the site, do I need to add anything else?